riscv/src/runtime.rs

```diff
@@ -92,65 +92,75 @@ impl Runtime {
         // Base submachines
         // TODO: can/should the memory machine be part of the runtime also?
         r.add_submachine(
-            "std::machines::binary::Binary",
+            "std::machines::binary_bb::Binary16",
```

Comment:
It would be better to name binary_bb instead of binary16 since it will also work for other field elements, and we already have a `Binary16` machine

Reply:
I will change the machine to named like that so that they can merge it.